user/sonarcloud.md

```diff
@@ -76,23 +76,14 @@ SonarCloud can inspect internal pull requests of your repository and write comme
 
 > For security reasons, this advanced feature works only for **internal** pull requests. In other words, pull requests built from forks won't be inspected.
 
-To activate analysis on pull requests, you need to follow those extra steps:
+To activate analysis on pull requests, you simply need to [install the SonarCloud application](https://github.com/apps/sonarcloud) on your GitHub organization(s).
```

Comment:
We try to avoid using words like "simply" in our docs.

Reply:
Feedback applied, thanks!